cities/weather_api_utils.py

```diff
@@ -0,0 +1,11 @@
+import main
+
+
+async def get_weather(city_name: str):
+    response = await main.get_http_client().get(
+        f"{main.WEATHER_API_URL}"
+        f"?key={main.WEATHER_API_KEY}&q={city_name}&aqi=no"
+    )
+    if response.status_code == 200:
+        json = response.json()
+        return json.get("current").get("temp_c")
```

Comment:
Consider adding error handling for the HTTP request. If the request fails or the response does not contain the expected data, it could lead to runtime errors. You might want to handle exceptions such as `httpx.HTTPStatusError` or check if the JSON response contains the expected structure before accessing it.